skyvern/core/script_generations/real_skyvern_page_ai.py

```
@@ -17,6 +17,8 @@
 from skyvern.forge.sdk.api.llm.schema_validator import validate_and_fill_extraction_result
 from skyvern.forge.sdk.core import skyvern_context
 from skyvern.forge.sdk.schemas.totp_codes import OTPType
+from skyvern.schemas.workflows import BlockStatus
+from skyvern.services import script_service
 from skyvern.services import script_service
```

Ruff (0.14.7)

Redefinition of unused `script_service` from line 21

Remove definition: `script_service`

(F811)